### PR TITLE
Add test and ci on github action

### PR DESCRIPTION
**Pull request by Ameausoone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,11 @@ This action slug and expose some github variables.
     echo ${{ env.GITHUB_HEAD_REF_SLUG }}
     echo ${{ env.GITHUB_BASE_REF_SLUG }}
 ```
+
+# Test
+
+To run test locally, run:
+
+```shell script
+docker run -w /workdir -v $(pwd):/workdir dduportal/bats:latest ./
```
Comment (rlespinasse) [inline README.md]: Can you put this command line under a `test` task in a `Makefile` file?

```
.PHONY: tests
tests:
	docker run -w /workdir -v $(shell pwd):/workdir dduportal/bats:latest ./
```

So we can just run `make tests`

File: entrypoint-test.bats [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+@test "slug" {
```
Comment (rlespinasse) [inline entrypoint-test.bats]:
```suggestion
@test "slug_ref" {
```

**Review by rlespinasse — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ameausoone) [reply README.md]: done

**Review by rlespinasse — APPROVED** [review]

**Pull request merged**